src/ocamlutil/pretty.ml

```diff
@@ -241,33 +249,33 @@ type align =
                                    taking the break associated with this
                                    alignment mark. If this is 0, then there
                                    is no break associated with the mark *)
-      mutable isTaken: bool ref; (* If breakGain is > 0 then this is a ref
+      mutable isTaken: bool refDLS; (* If breakGain is > 0 then this is a ref
                                     cell that must be set to true when the
                                     break is taken. These ref cells are also
                                     int the "breaks" list  *)
-            deltaFromPrev: int ref; (* The column of this alignment mark -
+            deltaFromPrev: int refDLS; (* The column of this alignment mark -
                                        the column of the previous mark.
                                        Shared with the deltaToNext of the
                                        previous active align  *)
-             deltaToNext: int ref  (* The column of the next alignment mark -
+             deltaToNext: int refDLS  (* The column of the next alignment mark -
                                       the columns of this one. Shared with
                                       deltaFromPrev of the next active align *)
     }
 
 (* We use references to avoid the need to pass data around all the time *)
-let aligns: align list ref =  (* The current stack of active alignment marks,
+let aligns: align list refDLS =  (* The current stack of active alignment marks,
```

Comment:
Here `aligns` is DLS, but also every field of `align` is made DLS. The latter seems excessive then, or would individual elements actually be exchanged between domains somehow?